graylog2_handler/__openerp__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "GrayLog2 Handler",
+    "version": "8.0.0.0.1",
+    "author": ("Management and Accounting On-line, "
+               "Odoo Community Association (OCA)"),
+    "website": "https://github.com/OCA/server-tools/",
+    "summary": "Provides ability to send log messages to graylog2 server",
+    "category": "Added functionality",
+    "depends": [
+        'base',
+    ],
+    "active": True,
```

Comment:
This key name is deprecated, and afaik it is teh equivalent to "auto_install", which we don't want.
Probably you wanted to use `"installable": True`